go/materialize/driver/jsonimage/driver.go

```diff
@@ -199,12 +199,16 @@ func invokeConnector(
 ) error {
 
 	/*
-		return exec.Command(
-			"docker",
-			"run",
-			"--rm",
-			c.Image,
-		)
+	   return exec.Command(
+	       "docker",
+	       "run",
+	       // Without the --interactive flag, running connectors with docker-for-mac will sometimes
```

Comment:
I added it here just so we don't forget about it when we come back and work on this code again.